docs/rules/graphic-elements.md

```diff
@@ -39,7 +39,7 @@ Field natural wonder elements:
 <p align="center"><img src="../../ressources/img/map/field-wonder-everest.png"> <img src="../../ressources/img/map/field-wonder-galapagos.png"> <img src="../../ressources/img/map/field-wonder-kilimanjaro.png"> <img src="../../ressources/img/map/field-wonder-messa.png"> <img src="../../ressources/img/map/field-wonder-pantanal.png"> <img src="../../ressources/img/map/field-wonder-volcanic.png"> </p align="center">
 
 Barbarians:
-<p align="center"><img src="../../ressources/img/map/barbarians.png"></p align="center"> 
+<p align="center"><img src="../../ressources/img/map/barbarians.png"> <img src="../../ressources/img/map/barbarian-hutte.png"></p align="center"> 
```

Comment:
Currently, images do not appear when giving relative paths. You can give just the name of the image, it will not appear in the markdown viewer but will appear on the website.
Exemple:
src="barbarian-hutte.png"